config.schema.yml

```diff
@@ -1,17 +0,0 @@
----
```

Comment:
I don't know how I missed this the first time around. Sorry, I should have identified it when reviewing. 

There was some inconsistency in the past with what ST2 allowed, for `yml` vs `yaml`. But it should be all `.yaml` now.